package.json

```diff
@@ -16,36 +16,34 @@
     "@react-navigation/native": "^6.0.8",
     "@react-navigation/native-stack": "^6.5.0",
     "@testing-library/react-native": "^9.0.0",
-    "@types/react-test-renderer": "^17.0.1",
     "expo": "^44.0.0",
     "expo-app-loading": "~1.3.0",
     "expo-cli": "^4.13.0",
     "expo-modules-core": "^0.6.5",
     "expo-splash-screen": "~0.14.1",
     "expo-status-bar": "~1.2.0",
     "expo-updates": "~0.11.6",
-    "jest-expo": "^44.0.1",
+    "html-entities": "^2.3.2",
     "prop-types": "^15.8.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-native": "^0.67.3",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "react-native": "0.64.3",
```

Comment:
Should these dependencies be downgraded from ^17.0.2 to 17.0.1 and ^0.67.3 to 0.64.3 respectively?

Reply:
If this resolves the expo start dep incompatibility warnings, then great.

Reply:
Downgraded to expo compatibility.